final_prep/README.md

```diff
@@ -44,31 +44,31 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- ![week 1](https://user-images.githubusercontent.com/16493270/164334283-c56073b1-0520-447a-987d-f2b5ff4e78cc.png)`Add Week 1 Screenshot Here`
+- ![week 2](https://user-images.githubusercontent.com/16493270/164334357-d50bc43b-ebe1-469e-bef0-ab421030f5f5.png)`Add Week 2 Screenshot Here`
+- ![week 3](https://user-images.githubusercontent.com/16493270/164334416-86bb0d2f-766d-488b-8002-c6d391d00a57.png)`Add Week 3 Screenshot Here`
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
 - [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] Add link to your gist here:[mentor gist](https://gist.github.com/mkbonini/b66451816689771e9f966298e7c1e528)
```

Comment:
Nice intro message! Have you thought about how you'd like to work with your mentor? How you like to get feedback from a mentor? What you'll need from them when you are in the depths of "the struggle"?